### DISCUSSION TITLE
gradle vs maven

### DISCUSSION THREAD

**Discussion by haton0912** [community/community#69287] [Codespaces]
### Select Topic Area

Question

### Body

Hi,
I am using maven for microservice project.
My friend suggest me to use gradle for project.
If anyone let me know about gradle advantage over maven.

Kind Regard,
Javan

**Comment by krobinsmith** [comment]
Hi Javan,

Gradle is based on developing domain-specific language projects. Maven is based on developing pure Java language-based software. It uses a Groovy-based Domain-specific language(DSL) for creating project structure. It uses Extensible Markup Language(XML) for creating project structure.

While Maven is the name for both a build tool and a repository type, Gradle ist just a build tool. It supports both Maven and Ivy repositories.

I hope this helps you well.

**Reply by haton0912** [reply]
Thank you so much, Robin!

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
